src/main/java/nextstep/subway/line/dto/LineResponse.java

```diff
@@ -1,29 +1,41 @@
 package nextstep.subway.line.dto;
 
 import nextstep.subway.line.domain.Line;
+import nextstep.subway.section.domain.Section;
+import nextstep.subway.section.domain.Sections;
+import nextstep.subway.station.domain.Station;
 
 import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
 
 public class LineResponse {
     private Long id;
     private String name;
     private String color;
     private LocalDateTime createdDate;
     private LocalDateTime modifiedDate;
+    private List<Station> stations;
```

Comment:
아래처럼 `StationResponse`를 가지고 있는게 좋겠네요 :)

```suggestion
    private List<StationResponse> stations;
```